src/lib/definition-parser.ts

```diff
@@ -365,6 +366,12 @@ async function getModuleInfo(directory: string, folderName: string, allEntryFile
 	}
 }
 
+/** Given "foo/bar/baz", return "foo". */

```

Comment:
that is true as long as scoped packages are not supported. `@foo/bar/baz`'s root is `@foo/bar`.